src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyFlowGraphOperation.cs

```diff
@@ -64,6 +65,40 @@ public override async Task<int> ExecuteAsync()
                     flowGraph.PruneGraph(node => IsInterestingNode(targetChannel, node), edge => IsInterestingEdge(edge));
                 }
 
+                // For each node, determine number of changes that are made for a change starting at that node.
```

Comment:
Are the changes to this file expected in this change? seem like something else

Reply:
Oh interesting...I wonder how this got in here.  Remove.  It was related to some calculation I was doing

Reply:
Removed.